### PR TITLE
New tokens for gcds-card

### PR DESCRIPTION
**Pull request by ethanWallace**
# Summary | Résumé

New component tokens for `gcds-card` component.

Built for use with https://github.com/cds-snc/gcds-components/pull/183

### CODE REVIEW

File: tokens/components/card/tokens.json [hunk 1]
```diff
@@ -0,0 +1,128 @@
+{
+  "card": {
+    "backgroundColor": {
+      "value": "{color.grayscale.0.value}",
+      "type": "color"
+    },
+    "border": {
+      "value": "{border.width.sm.value} solid {color.grayscale.100.value}",
```
Comment (melaniebmn) [inline tokens/components/card/tokens.json:8]: More of a design comment but: can we change this to use our default border token? `grayscale.100` does not pass the contrast ratio for borders and also deviates from what we defined as the default border.

**Review by melaniebmn — CHANGES_REQUESTED** [review]
Added a question about the border colour that we should address before merging.

File: tokens/components/card/tokens.json (hunk 1 above)
Comment (dylanzheng94) [reply tokens/components/card/tokens.json:8]: Updated the Figma component to use the default border token.

Comment (melaniebmn) [reply tokens/components/card/tokens.json]: Thank you!

**Review by melaniebmn — APPROVED** [review]
LGTM

**Pull request merged**